phpseclib/Crypt/Base.php

```diff
@@ -645,16 +626,19 @@ function setPassword($password, $method = 'pbkdf2')
      * strlen($plaintext) will still need to be a multiple of the block size, however, arbitrary values can be added to make it that
      * length.
      *
-     * Note: Could, but not must, extend by the child Crypt_* class
-     *
      * @see Crypt_Base::decrypt()
      * @access public
      * @param String $plaintext
-     * @return String $cipertext
+     * @return String $ciphertext
+     * @internal Could, but not must, extend by the child Crypt_* class
      */
     function encrypt($plaintext)
     {
-        if ($this->engine == CRYPT_MODE_MCRYPT) {
+        if ($this->paddable) {
```

Comment:
Not sure how this change is related to "add setPreferredEngine". Seems to be unrelated cleanup.